idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -161,29 +165,35 @@ class PackedColumnData(UnitModelBlockData):
         domain=float,
         description="fixed column pressure in Pa",
         doc="Fixed column operating pressure in Pa"))
+    CONFIG.declare("fix_column_pressure_drop", ConfigValue(
```

Comment:
Similarly, isn;t this just a design condition the user should be fixing (or providing a sub-model for)?